webportal/package.json

```diff
@@ -45,6 +46,7 @@
     "html-webpack-plugin": "2.30.1",
     "joi": "13.0.2",
     "jquery": "3.2.1",
+    "jquery.ajax-cross-origin": "1.1.0",
```

Comment:
pls use `npm run yarn add pkg` instead of `npm install pkg`

Reply:
#Solved.